charts/openshift-etcd-backup/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Default values for openshift-etcd-backup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+backup:
+  # Sub directory path
+  subdir: "/"
+  # Directory name of single backup
+  dirname: "+etcd-backup-%F-%H-%M-%S"
+  # expiretype could be days (keep backups newer than backup.keepdays,
+  # count (keep a number of backups with backup.keepcount),
+  # never (do not expire backups, keep all of them)
+  expiretype: "days"
+  # retention period
+  keepdays: "30"
+  # count retetion if expiretype set to count
+  keepcount: "10"
+  # backup schedule
+  schedule: "0 0 * * *"
+
+persistence:
+  # Set reclaim policy (Retain or Delete)
+  reclaimPolicy: Retain
+  # Define the storage size
+  capacity: 10Gi
+  nfs:
+    # Enable nfs backend storage
+    enabled: false
+    # NFS server name or IP
+    server: example.com
+    # NFS server path
+    path: "/etcd-backups"
+  hostPath:
+    # Enable hostPath
+    enabled: true
+    # hostPath existing path on host
+    path: "/opt/etcd-backups"
```

Comment:
[`/opt/`](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s13.html) is probably not the best path here, use something below [`/mnt`](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#mntMountPointForATemporarilyMount) (in case it's temporary) or something below [`/var/opt/`](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s12.html).

Reply:
thanks for the heads-up on that